docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,285 @@
+# HVAC_SystemZoneAssignment - Rule 11-10
+
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 11-10  
+
+**Rule Description:** "The service water heating systems type in the baseline building design shall match the minimum efficiency requirements in Section 7.4.2"  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b  
+
+**Evaluation Context:** Each SWH bat  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_SHW_equipment_connected_to_use_type()
+2. get_SHW_types_and_SHW_use()
+
+**Applicability Checks:**
+- Each SHW use type is applicable if there are SHW loads
```

Comment:
General comment to use the term "SWH BAT" instead of use type. Please apply throughout the RDS.